src/routes/v2/sgidAuth.ts

```diff
@@ -149,6 +250,10 @@ export class SgidAuthRouter {
       "/verify-login",
       attachReadRouteHandlerWrapper(this.handleSgidLogin)
     )
+    router.post(
+      "/verify-multiuser-login",
```

Comment:
our path structure isn't ideal but this is something that can be fixed down the line. this should instead be `/verify/multi/login` or something

Reply:
48a15e38